torch/distributed/_tensor/dispatch.py

```diff
@@ -25,7 +25,7 @@
 
 
 def wrap(res: object, spec: OutputSpecType) -> object:
-    if isinstance(res, torch.Tensor):
+    def to_dt(res, spec):
```

Comment:
Hey @wanchaol , is this change OK? fused adam returns tuple of lists, so we need this.